### PR TITLE
Use one default viewport

### PR DESCRIPTION
**Pull request by stefanhendriks**
Solves #4

### CODE REVIEW

File: src/main/java/com/fundynamic/d2tm/game/PlayingState.java [hunk 1]
```diff
@@ -102,6 +95,7 @@ void render() throws SlickException {
         }
 
         void update() {
+            // use events?
```
Comment (arjenvanderende) [inline src/main/java/com/fundynamic/d2tm/game/PlayingState.java]: Instead of adding a comment, maybe open a Github issue instead?

**Comment by arjenvanderende** [comment]
Looks good to me :+1:

File: src/main/java/com/fundynamic/d2tm/game/PlayingState.java (hunk 1 above)
Comment (stefanhendriks) [inline src/main/java/com/fundynamic/d2tm/game/PlayingState.java]: You mean like #3  ? ;-)

**Comment by stefanhendriks** [comment]
Glad you like it ;-)

File: src/main/java/com/fundynamic/d2tm/game/PlayingState.java (hunk 1 above)
Comment (arjenvanderende) [inline src/main/java/com/fundynamic/d2tm/game/PlayingState.java]: Exactly like #3 :wink:

**Pull request merged**